### ISSUE TITLE
Moving the autoplotter into a separate process.

### ISSUE THREAD

**Issue by bjoernenders** [ptycho/ptypy#171]
Having Autoplotter running in a thread off the master process should be avoided. The autoplotter occasionally doesn't quit and continues to linger on the server, obstructing ports and requiring manual kill.

**Comment by alexbjorling** [comment]
Yes, our user instructions for how to reconstruct start with `killall python`, very elegant.
